packages/main/src/components/AnalyticalTable/__snapshots__/AnalyticalTable.test.tsx.snap

```diff
@@ -32,7 +32,7 @@ exports[`AnalyticalTable Alternate Row Color 1`] = `
         style="width: 600px;"
       >
         <div
-          class="resizer-0-2-31"
+          class="resizer-0-2-32"
```

Comment:
why do we even have rule counters here?

Reply:
There was no name specified in the `createUseStyles` options which was the reason for this behavior